Website/src/rendering/src/assets/css/rest.css

```diff
@@ -77,12 +77,11 @@
   content: '';
   width: 50%;
   height: 100%;
-  background:
-    -webkit-linear-gradient(
-      left,
-      rgba(255, 255, 255, 0) 0%,
-      rgba(255, 255, 255, 0.3) 100%
-    );
+  background: -webkit-linear-gradient(
```

Comment:
The build failed with a linting error. Could you please have a look at it? Thanks